src/components/animated/_trace.less

```diff
@@ -0,0 +1,8 @@
+.trace-btn{
+    border:5px solid black;
+    padding: 1.5rem;
```

Comment:
Please remove the width and the padding as the sizing of the buttons shouldn't change